docs/getstarted/step_six.md

```diff
@@ -72,13 +72,12 @@ If you don't already have a terminal open, open one now:
 
     The format for the login command is:
 
-        docker login --username=yourhubusername --email=youremail@company.com
+        docker login --username=yourhubusername --password=fake123
```

Comment:
I think we'd better remove the `--password` flag here; in general, it's not recommended to used that option, because it'll leak your password (e.g. In your shell history)